WcaOnRails/Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'sdoc', group: :doc
 gem 'dotenv-rails', require: 'dotenv/rails-now'
 gem 'envied'
 gem 'seedbank'
+gem 'seed_dump'
```

Comment:
This gem feels like it should not be installed in production. Maybe move it to the development group? However, it looks like the reason you added it may be superceded by Jonatan's astute observation that it's not needed =)